### ISSUE TITLE
Malfunction with bulleted list

### ISSUE THREAD

**Issue by mbenningfield1** [NaturalDocs/NaturalDocs#36]
Documenting a C library, and this comment section:

    /* Function: SetEdgeFalloff
    **
    ** Sets the falloff value at the edge transition.
    **
    **  Params:
    **    edgeFalloff  - The falloff value at the edge transition.
    **
    ** Notes:
    ** The falloff value is the width of the edge transition at either edge of the
    ** selection range. By default, there is an abrupt transition between the
    ** values from the two source modules at the boundaries of the selection range.
    ** For example, if the selection range is 0.5 to 0.8, and the edge falloff
    ** value is 0.1, then the GetValue() function outputs:
    **
    **    - if the output value from the control module is less than 0.4
    **      (0.5 - 0.1), the result is the output value from the source module
    **      at index 0.
    **
    **    - if the output value from the control module is between 0.4 (0.5 - 0.1)
    **      and 0.6 (0.5 + 0.1), the result is a linear blend between the output
    **      values from the two source modules.
    **
    **    - if the output value from the control module is between 0.6 (0.5 + 0.1)
    **      and 0.7 (0.8 - 0.1), the result is the output value from the source
    **      module with an index value of 1.
    **
    **    - if the output value from the control module is between 0.7 (0.8 - 0.1)
    **      and 0.9 (0.8 + 0.1), the result is a linear blend between the output
    **      values from the two source modules.
    **
    **    - if the output value from the control module is greater than 0.9
    **      (0.8 + 0.1), the result is the output value from the source module
    **      with an index value of 0.
    */

Comes out looking like this:
![malf](https://user-images.githubusercontent.com/22527582/57965622-57bbba80-790c-11e9-8ed9-9c96122c08c1.png)

If I remove the intervening blank lines, it comes out ok, but makes the comments in the header file virtually unreadable. Also, this comment:

    /* Type: SelectModule
    **
    ** Noise module that outputs the value selected from one of two source modules
    ** chosen by the output value from a control module.
    **
    ** Unlike most other noise modules, the index value assigned to a source module
    ** determines its role in the selection operation:
    **
    **    - Source module 0 outputs a value.
    **
    **    - Source module 1 outputs a value.
    **
    **    - Source module 2 is known as the 'control module'.  The control module
    **      determines the value to select.  If the output value from the control
    **      module is within a range of values known as the 'selection range', this
    **      noise module outputs the value from the source module with an index
    **      value of 1.  Otherwise, this noise module outputs the value from the
    **      source module with an index value of 0.
    **
    ** To specify the bounds of the selection range, call the SetBounds() function.
    **
    ** An application can pass the control module to the SetControlModule()
    ** function instead of the SetSourceModule() function. This may make the
    ** application code easier to read.
    **
    ** By default, there is an abrupt transition between the output values from the
    ** two source modules at the selection-range boundary. To smooth the transition,
    ** pass a non-zero value to the SetEdgeFalloff() function. Higher values result
    ** in a smoother transition.
    **
    ** This noise module requires three source modules.
    */

Comes out just fine, like this:

![fine](https://user-images.githubusercontent.com/22527582/57965689-268fba00-790d-11e9-9f79-c6db66f45a0a.png)

So I don't know if this is a bug or if there is something I'm doing right from one comment to the other, although they look identically formatted to me.

**Comment by mbenningfield1** [comment]
Problem solved by removing the spacing around the "+" and "-" operators. I guess they were being interpreted as list signifiers.

**Comment by NaturalDocs** [comment]
Yes, in two of what should be bullets the " - " is causing it to be interpreted as a definition list instead, like "param1 - description".  I'm going to reopen this because I don't like this behavior and can probably fix it by moving the bullet detection before the definition list detection.
